orca-spring-batch/src/main/java/com/netflix/spinnaker/orca/batch/stages/ParallelDeployStageDefinitionBuilder.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.batch.stages;
+
+import com.netflix.spinnaker.orca.DefaultTaskResult;
+import com.netflix.spinnaker.orca.ExecutionStatus;
+import com.netflix.spinnaker.orca.Task;
+import com.netflix.spinnaker.orca.TaskResult;
+import com.netflix.spinnaker.orca.batch.StageBuilder;
+import com.netflix.spinnaker.orca.batch.StageBuilderProvider;
+import com.netflix.spinnaker.orca.pipeline.BranchingStageDefinitionBuilder;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import com.netflix.spinnaker.orca.pipeline.model.SyntheticStageOwner;
+import org.springframework.batch.core.job.builder.FlowBuilder;
+import org.springframework.batch.core.job.flow.Flow;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ParallelDeployStageDefinitionBuilder extends ParallelStageDefinitionBuilder {
+  public ParallelDeployStageDefinitionBuilder(BranchingStageDefinitionBuilder delegate,
+                                              StageBuilderProvider stageBuilderProvider) {
+    super(delegate, stageBuilderProvider);
+  }
+
+  @Override
+  protected List<Flow> buildFlows(Stage stage) {
```

Comment:
This is what `ParallelDeployStage` is doing on `master` ... 

https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/ParallelDeployStage.groovy#L57